### DISCUSSION TITLE
Unable to see text property (other works) when included in webcomponent

### DISCUSSION THREAD

**Discussion by GalinhaLX** [mavoweb/mavo#904] [Q&A]
Hi

When text property is included in webcomponent with mv-autoedit class,
property doesn't display value

(It only works without mv-autoedit class)

I tried on my [codepen](https://codepen.io/galinhalx/pen/eYMLvBe) to apply z-index
[property="name"] {
	z-index: 10000;
}
without success

But I think it's something to do with height, because property...is in the html code

Does mavo needs to have a specific height declared for bodu html element at beginning?

**Comment by DmitrySharabin** [comment]
Hey there,

If I'm not mistaken, the issue you face is related to this one: #618. And it might be an issue with [Stretchy](https://stretchy.verou.me) (as Lea mentioned).

As a workaround I would suggest adding this CSS rule to your style sheet:

```css
[property] > .mv-editor {
	min-block-size: 2em !important;
}
```

Instead of `2em` you can use any value that makes sense for you. You can also be more specific with the selector to not target all the editors.

I suggest using `!important` (even though it's a dead sin) to be more specific than the inline rule (`style="height: 0;"`) Stretchy adds to the editor.

**Comment by GalinhaLX** [comment]
Thanks @DmitrySharabin

Unfortunately it doesn't solve my problem:

As I have different texts, with different sizes... at the end, all looks strange :(
-> Some will wrap content, some edit 'boxes will be bigger than content, others smaller...

So I get this kind of result (in same previous [codepen](https://codepen.io/galinhalx/pen/eYMLvBe))

<img width="411" alt="image" src="https://user-images.githubusercontent.com/89095737/184396746-1c798aff-8062-440f-9820-873116c7f0c2.png">



Is 'mv-edit-elementype' implemented in Mavo? Not sure to fully understand @LeaVerou 's comment
